src/components/TradesWidget/index.tsx

```diff
@@ -48,13 +49,28 @@ function csvTransformer(trade: Trade): CsvColumns {
     fillPrice,
     sellAmount,
     buyAmount,
+    orderSellAmount,
     timestamp,
     txHash,
     eventIndex,
     orderId,
     batchId,
   } = trade
 
+  const limitPriceStr = limitPrice ? formatPrice({ price: invertPrice(limitPrice), decimals: 8 }) : 'N/A'
+  const inverseLimitPriceStr = limitPrice ? formatPrice({ price: limitPrice, decimals: 8 }) : 'N/A'
+  const fillPriceStr = formatPrice({ price: invertPrice(fillPrice), decimals: 8 })
+  const inverseFillPriceStr = formatPrice({ price: fillPrice, decimals: 8 })
+
+  const fillPercentage =
+    orderSellAmount && trade.type != 'liquidity'
+      ? new BigNumber(sellAmount.toString()).dividedBy(new BigNumber(orderSellAmount.toString())).toString(10)
```

Comment:
that's long for just dividing two numbers. Is there not a util for this?

Reply:
It's because I'm converting the amounts from BNs to string then to BigNumber in order to have a decimal precision result.
Sure, I'll create an untils function for that.